umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md

```diff
@@ -22,7 +25,45 @@ In the **Members** section, you can get an overview, view pending invites, and t
 
 <figure><img src="../../../.gitbook/assets/members.png" alt=""><figcaption></figcaption></figure>
 
-### MFA enforcement
+Members added to your ogranization can see different information about the organization based on the user group they are added to. Currently there are three different groups, **Read**, **Write** and **Admin**.
+Below you can see what each user group has access to under the organization they are a part of.
+
+Organization Members with **Admin Access** can do the following in the organization:
+
+* Update the organization information
+* Invite to the organization
+* Invite Users to project under the organization
```

Comment:
Hmm, am not sure if you are actually able to invite users to the projects from the organization section. Can you check with Søren to review this PR as well to confirm these details? 

Also, can you ask him as well what is the difference between users and members (as in some cases members are used an others users are used)? I am not sure I understand what is the difference between them in terms of organization 🤔

Reply:
Already checked with Søren, and they can indeed invite users to projects :D 

Yeah, I understand the confusion, and I have talked with Søren about this in the past 🙈

Users are invited to cloud projects, and members is when you invite to an organization... In the end it is just a matter of terms, and as long as it is like that in the cloud portal it should also be this way in the docs 😊 I do hope it makes sense :D